app/views/guides/community/before_asking.html.erb

```diff
@@ -18,7 +18,7 @@
     <h3>Break Down the Problem</h3>
 
     <p>
-      Break down the problem into small pieces and write it out in pseudo code. If you would like a detailed explanation of this step, read our <a href="lessons/foundations-problem-solving" target="_blank">Problem Solving lesson</a> with this specific problem in mind.
+      Break down the problem into small pieces and write it out in pseudo code. If you would like a detailed explanation of this step, read our <a href="../../lessons/foundations-problem-solving" target="_blank">Problem Solving lesson</a> with this specific problem in mind.
```

Comment:
Nit: A leading forward slash should be all thats needed. It will make it start from the root.

```suggestion
      Break down the problem into small pieces and write it out in pseudo code. If you would like a detailed explanation of this step, read our <a href="/lessons/foundations-problem-solving" target="_blank">Problem Solving lesson</a> with this specific problem in mind.
```